src/main/scala/fs2/aws/kinesis/RecordProcessor.scala

```diff
@@ -14,7 +14,7 @@ import scala.collection.JavaConverters._
   *  @constructor create a new instance with a callback function to perform on record receive
   *  @param cb callback function to run on record receive, passing the new CommittableRecord
   */
-private[kinesis] class RecordProcessor(cb: CommittableRecord => Unit) extends v2.IRecordProcessor {
+private[aws] class RecordProcessor(cb: CommittableRecord => Unit) extends v2.IRecordProcessor {
```

Comment:
why aws? this is a kinesis specific class

Reply:
i moved KinesisConsumerSpec to fs2.aws package and it is used there

Reply:
I can put it back to fs2.aws.kinesis and make it package private

Reply:
@dmateusp can you approve this PR?